rules/Php70/Rector/FuncCall/MultiDirnameRector.php

```diff
@@ -56,12 +56,14 @@ public function refactor(Node $node): ?Node
         $activeFuncCallNode = $node;
         $lastFuncCallNode = $node;
 
+        $shouldUpdate = false;
```

Comment:
In this rule we should only update the parameters if there was at least one inner call to the `dirname` function